demo/kitchen_sink/run.py

```diff
@@ -73,7 +73,7 @@ def fn(
             (".", 0),
         ]
         + [(f"test", x / 10) for x in range(-10, 10)],  # HighlightedText
-        json.loads(JSONOBJ),  # JSON
+        json.loads(JSONOBJ),  # JSONåå
```

Comment:
Weird symbols showing up at the end for me?

Reply:
whoops good catch